src/main/resources/schemas/workload.json

```diff
@@ -78,6 +78,11 @@
                       }
                     }
                   }
+                },
+                "connection_manager": {
+                  "type": "integer",
+                  "title": "Connection manager index",
+                  "description": "Positional index of the connection manager within the connections configuration"
```

Comment:
This description may need more details, for instance is it 0 indexed or hardcoded based on value in connection config?